ddtrace/tracer/textmap_test.go

```diff
@@ -232,17 +286,43 @@ func TestTextMapPropagatorTraceTagsTooLong(t *testing.T) {
 	assert.Nil(t, err)
 	sctx, ok := ctx.(*spanContext)
 	assert.True(t, ok)
-	child := tracer.StartSpan("test", ChildOf(sctx))
+	assert.Equal(t, "extract_max_size", sctx.trace.tags["_dd.propagation_error"])
+}
+
+func TestTextMapPropagatorInjectTraceTagsTooLong(t *testing.T) {
```

Comment:
Why aren't these just one test with sub-tests? Their naming is clearly hinting at it.